programs/merkle-distributor/src/lib.rs

```diff
@@ -294,6 +383,23 @@ pub struct MerkleDistributor {
     pub num_nodes_claimed: u64,
 }
 
+#[account]
+#[derive(Default)]
+pub struct Config {
+    pub distributor: Pubkey,
```

Comment:
do we need config.distributor?

Reply:
i guess having it is slightly more convenient if we wanna dump and correlate the data